pure-selenium/src/test/java/tests/CheckoutTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import org.example.pages.CartPage;
+import org.example.pages.CheckoutPage;
+import org.example.pages.InventoryPage;
+import org.example.pages.LoginPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class CheckoutTest extends AbstractTest{
+
+    @Test
+    public void testProceedToCheckout() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.open();
+        loginPage.login("standard_user", "secret_sauce");
+
+        InventoryPage inventoryPage = new InventoryPage(driver);
```

Comment:
You should return a new InventoryPage instance directly from the login method, It makes your code cleaner.

LoginPage loginPage = new LoginPage(driver);
InventoryPage inventoryPage = loginPage.login("standard_user", "secret_sauce");